ui/dialog/MintDialog.tsx

```diff
@@ -101,14 +178,13 @@ const MintDialog = ({ open, handleOpen }: MintDialogProps) => {
           Close
         </DialogFooterButton>
 
-        <DialogFooterButton
-          color="green"
-          onClick={() => {
-            alert(`You Minted ${mintedDegens} degens!`);
-          }}
-        >
-          Mint
-        </DialogFooterButton>
+        {
+          loading ?
+          <DialogFooterButton color="green"> Minting </DialogFooterButton>
+          :
+          <DialogFooterButton color="green" onClick={onMint}> Mint </DialogFooterButton>
+        }
+
```

Comment:
we could use tailwind to add some loading overlays or cool animations https://tailwindcss.com/docs/animation